### ISSUE TITLE
`redcap_file_repo_list()`

### ISSUE THREAD

**Issue by wibeasley** [OuhscBbmc/REDCapR#549]
requested by @agdamsbo, call REDCap's "Export a List of Files/Folders from the File Repository" API endpoint.

@agdamsbo, I assume this is the file tree you were referring to?

Right now, the new `redcap_file_repo_list_oneshot()` works as expected, but it doesn't show the whole repo structure, just one folder per call.  If I added something recursive to list the complete repo, would people use it?  We haven't really used REDCap's file repo much, and I'm not that familiar with the typical use cases.

![image](https://github.com/user-attachments/assets/c1b7b87f-9c96-4c57-9397-97b8f8d53398)

**Comment by wibeasley** [comment]
Right, now it looks like API import privileges are required, and I [posted this](https://redcap.vumc.org/community/post.php?id=243161) to the community.

**Comment by agdamsbo** [comment]
Wow. Swift response! This is something I have missed, and I will get around to testing. Thank you @wibeasley! 
In my group this would be used for easily including database documentation and reference papers.
